README.md

```diff
@@ -108,7 +108,7 @@ npm run build -- Debug
 ```
 NOTE: the build will take a while to complete. Depending on your processor and memory, it could potentially take a few hours.
 
-Brave staff may also want to try [Goma](https://github.com/brave/devops/wiki/Faster-browser-builds#goma) for faster builds.
+Brave staff may also want to try [Goma](https://github.com/brave/brave-browser/wiki/Brave-Developer-tips#speeding-up-builds) for faster builds.
```

Comment:
The correct link is https://github.com/brave/devops/wiki/Remote-Build-Execution

Reply:
`Brave staff may also want to try [RBE](https://github.com/brave/devops/wiki/Remote-Build-Execution) for faster builds.`

Reply:
@mihaiplesa your link does not work

Reply:
It is internal and only for Brave staff.

Reply:
@mihaiplesa your total wrong it is not for brave staff it is for chromium / google staff brave has only a user account because we like your project in general and we are open. There is a high chance that some one who contributes often to chromium project forks has a goma account is relative high. the scene is small ;)

Reply:
@mihaiplesa also there is a lot of brave goma related docs missing like mentioned here:

https://github.com/brave/brave-browser/issues/20492

Reply:
Goma has been deprecated by Google.

Reply:
@mihaiplesa oh i was not aware of that i am working on a alternativ build system at present also. I am lead of the Chromium Virtual Env Project that aims to build chromium / android in the browser for any arch and platform. via wasm and emulated cpu instructions in a distributed fashion via webrtc. 

i did not know that something else happens in between lol

Reply:
Please see:
https://groups.google.com/a/chromium.org/g/chromium-dev/c/rajt7THxIng/m/ZoDB54wQBAAJ
https://blog.engflow.com/2023/09/11/goma-is-gone-put-everything-into-reclient/#input-processing